src/test/java/com/google/api/generator/test/framework/Assert.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.test.framework;
+
+import java.nio.file.Path;
+import java.util.List;
+import junit.framework.AssertionFailedError;
+
+public class Assert {
+  public static void assertCodeEquals(Path goldenPath, String codegen) {
```

Comment:
Since this is a core, critical utility, consider adding a Javadoc so that future readers can understand this more easily.

Reply:
Added a JavaDocComment to first method and a simple LineComment for the second method.